src/store/modules/actions/staticData.js

```diff
@@ -29,6 +31,43 @@ export async function fetchUserInfo({ commit, state }, opts) {
   return user;
 }
 
+export async function fetchUserInfoByAddress({ commit, state }, address) {
+  let userInfo = {
+    displayName: address,
+    avatar: `https://avatars.dicebear.com/api/identicon/${address}.svg`,
+  };
+
+  let promise = state.fetching.userInfo[address];
+  if (promise) {
+    userInfo = await promise;
+    return userInfo;
+  }
+
+  try {
```

Comment:
`promise` might throw error
```suggestion
  try {
    let promise = state.fetching.userInfo[address];
    if (promise) {
      userInfo = await promise;
      return userInfo;
    }
```